api/client-server/event_context.yaml

```
@@ -34,6 +34,9 @@ paths:
         This API returns a number of events that happened just before and
         after the specified event. This allows clients to get the context
         surrounding an event.
+
+        *Note*: this endpoint supports lazy-loading. See `Filtering <#filtering>`_
```

Maybe the following style here and elsewhere?

```suggestion
        *Note*: This endpoint supports lazy-loading. See `Filtering <#filtering>`_
```

---

I have unreasonably strong opinions about this, mostly based on aesthetics and personal preference

---

Most usages of "Note:" that I can find in the spec follow that with a capital letter, which is personally what I'd expect, since "Note:" is not really part of the sentence that follows. Anyway, I would just suggest it would be nice to choose one approach or the other for grammatical consistency.

---

fair, I'll fix it.